src/panel/PanelManager.jsx

```diff
@@ -51,7 +51,7 @@ export default class PanelManager extends React.Component {
     window.times.appRendered = Date.now();
 
     listen('map_user_interaction', () => {
-      if (this.state.ActivePanel === PoiPanel && isMobileDevice()) {
+      if (isMobileDevice()) {
```

Comment:
I guess it's the wanted behavior even in direction panel ?

Reply:
Yep, that's what I explained in the comment :)

Reply:
ok, I missed it when re-reviewing